layouts/project/single.html

```diff
@@ -100,6 +97,23 @@ <h2>Funded by</h2>
 			</ul>
 		</div>
 		{{ end }}
-  	</div>
+		{{ if gt (len .Params.eclipse_projects) 0 }}
+		<div class="sidebar-panel margin-top-20 margin-bottom-20">
+			<h2>Eclipse Projects Contributions</h2>
+			<ul class="padding-left-0">
+				{{ range .Params.eclipse_projects }}
+				    {{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
+						<li class="vertical-align">
+							<a href="{{ .url }}" class="vertical-align flex-column">
```

Comment:
The class here doesn't need to be vertical-align flex-column, we can just use text-center